pkg/operators/manager.go

```diff
@@ -51,7 +50,11 @@ func RunClusterManagerOperator(ctx context.Context, controllerContext *controlle
 		return err
 	}
 
-	kubeInformer := informers.NewSharedInformerFactoryWithOptions(kubeClient, 5*time.Minute, informers.WithNamespace(helpers.ClusterManagerNamespace))
+	// kubeInformer is for 3 usages: configmapInformer, secretInformer, deploynmentInformer
+	// After we introduced detached mode, the hub components could be installed in a customized namespace.(Before that, it only inform from "open-cluster-management-hub" namespace)
+	// It requires us to add filter for each Informer respectively.
+	// TODO: Wathc all namespace may cause performance issue.
+	kubeInformer := informers.NewSharedInformerFactoryWithOptions(kubeClient, 5*time.Minute)
```

Comment:
note, this would need rbac update.

Reply:
I'm not sure about this one, since there is only `cluster_role` and `cluster_role_binding` exist in `/deploy/cluster-manager/config/rbac,  it seems already supported watching resources from all namespace before.